packages/functional-tests/tests/signin/signIn.spec.ts

```diff
@@ -0,0 +1,104 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { test, expect } from '../../lib/fixtures/standard';
+
+test.describe('signin here', () => {
+  test.beforeEach(async ({ target, pages: { login } }) => {
+    test.slow();
+  });
+
+  test('signin verified with incorrect password, click `forgot password?`', async ({
+    target,
+    page,
+    credentials,
+    pages: { login, resetPassword },
+  }) => {
+    await page.goto(target.contentServerUrl);
+    await login.setEmail(credentials.email);
+    await login.clickSubmit();
+    await login.setPassword('incorrect password');
+    await login.clickSubmit();
+
+    // Verify the error
+    expect(await login.getTooltipError()).toContain('Incorrect password');
+
+    //Click forgot password link
+    await login.clickForgotPassword();
+
+    //Verify reset password header
+    expect(await resetPassword.resetPasswordHeader()).toBe(true);
+  });
+
+  test('signin with email with leading/trailing whitespace on the email', async ({
+    target,
+    page,
+    credentials,
+    pages: { login },
+  }) => {
+    const emailWithSpace = '   ' + credentials.email;
+    await page.goto(target.contentServerUrl);
+    await login.fillOutEmailFirstSignIn(emailWithSpace, credentials.password);
+
+    // Verify the header after login
+    expect(await login.loginHeader()).toBe(true);
+
+    // Need to clear the cache to get the new email
+    await login.clearCache();
+
+    await page.goto(target.contentServerUrl);
+    const emailWithoutSpace = credentials.email + '  ';
```

Comment:
Very tiny nit, the variable name could be better. The email does have space, just at the end end.